src/NonPhpFile/NeonMethodCallRenamer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Core\NonPhpFile;
+
+use Rector\Core\Contract\Processor\FileProcessorInterface;
+use Rector\Core\ValueObject\NonPhpFile\NonPhpFileChange;
+use Rector\Renaming\Configuration\MethodCallRenameCollector;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+class NeonMethodCallRenamer implements FileProcessorInterface
```

Comment:
maybe it should be placed in rectorphp/rector-nette package ?

Reply:
Makes sense :+1:

Reply:
And what about the name? Maybe it is not neon feature but rather NetteDI... -> NetteDINeonMethodCallRenamer? :)

Reply:
What do you consider better?

Reply:
Now I like NetteDINeonMethodCallRenamer

Reply:
name changed